Stripe/PublicHeaders/StripeError.h

```diff
@@ -63,3 +63,9 @@ FOUNDATION_EXPORT NSString * __nonnull const STPIncorrectCVC;
     NSLocalizedString(@"There was an unexpected error -- try again in a few seconds", @"Unexpected error, such as a 500 from Stripe or a JSON parse error")
 #define STPCardErrorProcessingErrorUserMessage                                                                                                                 \
     NSLocalizedString(@"There was an error processing your card -- try again in a few seconds", @"Error when there is a problem processing the credit card")
+
+@interface STPError : NSError
```

Comment:
I may be missing something, but it looks like we never create any instances of `STPError`. Can we just extend `NSError` instead of defining a new type?